tests/generator_test.py

```diff
@@ -433,13 +433,14 @@ def test_simple_conditional(
         test_file_path = qir_files_dir / simple_conditional_file_name
 
         with open(test_file_path, "r") as input_file:
-            exp_data = input_file.readlines()
+            exp_data = input_file.read()
 
         for line in data:
             assert (
                 line in exp_data
             )  # Identical up to some ordering of the function declarations.
 
+    @pytest.mark.skip(reason="Disable work around non-simple circuits in CircBox.")
```

Comment:
Is this something you will be returning to? Maybe make a Jira task?

Reply:
Well, hopefully yes but it would certainly need to think about the non-simplicity of `CIrcBox` in `pytket`.

Reply:
In the Jira board [here](https://cqc.atlassian.net/browse/TKET-2662?atlOrigin=eyJpIjoiNTk2ZDI1OGJlMGY4NDdmM2JjYTIxMTdhOTc5NjYzOTMiLCJwIjoiaiJ9)

Reply:
Thanks. Could you describe what this means in terms of pytket-pyqir functionality?

- What pytket-pyqir functionality is missing because of the limitations of the pytket `CircBox` API?
- What change in pytket is needed to unblock this?

Reply:
Sure, I can try my best to describe that.